doc/source/contributing/contributing_prs.md

```diff
@@ -235,31 +236,33 @@ They are executed on the following order: clear-cache, build, test, docs, shell,
 
 ## Testing in Docker
 Testing the code using docker consists of executing the same file 3 times (you may need root privileges to run it).
-First run `./scripts/container.sh --clear-cache`. Then run `./scripts/container.sh --build`. This starts a local docker image called `arviz`. Finally run the tests with `./scripts/container.sh --test`. This should be quite close to how the tests run on TravisCI.
+First run `./scripts/container.sh --clear-cache`. Then run `./scripts/container.sh --build`. This starts a local docker image called `arviz`. Finally run the tests with `./scripts/container.sh --test`. This should be quite close to how the tests run on [TravisCI](https://travis-ci.org/).
```

Comment:
Minor nit: tests no longer run on Travis. General comment, now that you are modifying this, I would be much more explicit about what each flag does. as an example `--build` only needs to be executed once (or once in a while to update the docker image), **not** every time we want to test or build the docs. I am not sure this is clear to newcomers by reading this alone.

Reply:
Yeah, for docs testing we need to use the `--docs` flag. It is stated in the [Building documentation with Docker](https://arviz-devs.github.io/arviz/contributing/contributing_prs.html#building-documentation-with-docker) section. 

Here, do I need to make any changes? Should I remove the TravisCI line?

Reply:
> Should I remove the TravisCI line?

Travis has to be deleted because it is very outdated at this point. 

> Here, do I need to make any changes?

In general when reading the file, is it clear to you that you have to first build the docker image, then run the tests, or build the docs, or open a shell... and that you don't need to run build every time you do one of these? Was it clear to you when you set up docker? Is it clear that you have to rebuild the docs every time you make a change? Things like that. I think some changes would improve the doc but I'm not sure how many changes nor which ones

Reply:
No, it was not clear, and yeah, I thought about changing the docs section, then the thought that we are going to add sections like "How to read/write ArviZ docs" changed my mind. That might be adding the details in that section and adding its link here would be fine. But yes we can add the details here as well. What do you say?

For testing in Docker, we should change this part. Currently, it is giving the feeling that we need to build every time we want to test. But as you are saying that we need to run the `build` once, and only the `test` command again and again for testing. Right?

Reply:
Build needs to be executed only "once". It creates the docker image that can then be used to run any of the other commands as many times as desired. It only needs to be reexecuted to update the docker image if for example there have been new releases that we want to incorporate into the image like xarray or sphinx. 

The only command that is a bit special is test. IIUC, you can test with docker as many times as you want but if you test with docker and also locally you should run the clear cache command before every docker test because the cache generated locally is not valid for docker (and maybe viceversa too?)

The need for rebuilding the docs after every change is not docker specific so it doesn't need to be here, especially if it's already in the general doc guide, but I think it won't hurt to repeat that. Everything else does need to be here though because it's docker specific.